content/en/docs/reference/command-line-tools-reference/kubelet-tls-bootstrapping.md

```diff
@@ -186,18 +189,30 @@ roleRef:
   apiGroup: rbac.authorization.k8s.io
 ```
 
-Deleting the binding will prevent the node from renewing its client credentials, effectively
-removing it from the cluster once its certificate expires.
+Deleting the binding will prevent the node from renewing its client credentials,
+effectively removing it from the cluster once its certificate expires.
 
 ## kubelet configuration
-To request a client certificate from kube-apiserver, the kubelet first needs a path to a kubeconfig file that contains the
-bootstrap authentication token. You can use `kubectl config set-cluster`, `set-credentials`, and `set-context` to build this kubeconfig. Provide the name `kubelet-bootstrap` to `kubectl config set-credentials` and include `--token=<token-value>` as follows:
+
+To request a client certificate from kube-apiserver, the kubelet first needs a
+path to a kubeconfig file that contains the credentials for the identity that it
+will use to bootstrap its individual node identity.
+
+If you are using a bootstrap token, you can use `kubectl config set-cluster`,
+`set-credentials`, and `set-context` to build this kubeconfig.  Provide the name
+`kubelet-bootstrap` to `kubectl config set-credentials` and include
+`--token=<token-value>` as follows:
 
 ```
 kubectl config set-credentials kubelet-bootstrap --token=${BOOTSTRAP_TOKEN} --kubeconfig=bootstrap.kubeconfig
 ```
 
-When starting the kubelet, if the file specified by `--kubeconfig` does not exist, the bootstrap kubeconfig is used to request a client certificate from the API server. On approval of the certificate request and receipt back by the kubelet, a kubeconfig file referencing the generated key and obtained certificate is written to the path specified by `--kubeconfig`. The certificate and key file will be placed in the directory specified by `--cert-dir`.
+When starting the kubelet, if the file specified by `--kubeconfig` does not
+exist, the bootstrap kubeconfig is used to request a client certificate from the
```

Comment:
mention that bootstrap kubeconfig is specified via `--bootstrap-kubeconfig` flag